ops/_tracing/buffer.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+"""Buffer for tracing data."""
```

Comment:
I just want to confirm: there's no existing buffer handling in the opentelemetry libraries, correct?

Reply:
That's correct, there's not buffering to a file.
opentelemetry-sdk has a BatchSpanProcessor that effectively buffers N spans in memory.